benchmark/src/main/resources/ai/timefold/solver/benchmark/impl/report/chart-line.js.ftl

```diff
@@ -34,7 +31,7 @@ var ${chartId} = new Chart(document.getElementById('${chartId}'), {
                     borderWidth: 1
                   </#if>,
                   data: [
-                    <#list dataset.data() as datum><#if datum??>${datum?cn}</#if><#sep>, </#sep></#list>
+                    <#list chart.points(dataset.label()) as datum>{x: ${datum.key()?cn}, y: ${datum.value()?cn}}<#sep>, </#sep></#list>
```

Comment:
If I may recommend - don't touch the format of JS files.

I spent days tweaking chart.js to avoid all of its bugs and quirks. Sometimes it required weird choices in data set format. Unless you want to manually check every single supported chart, including tooltips, scaling and different browsers, I recommend not touching the data format.